sessions/http-api-docs-that-work.md

```diff
@@ -0,0 +1,17 @@
+# Building HTTP API Docs that Work
+
+The HTTP API is arguably our most important API — the primary recommended use of IPFS is for people to run a daemon in one process and control it via the HTTP API in another — so it needs to be well documented. Let’s figure out how to fix that.
+
+**Activity/Agenda:**
+1. Introduce some of the primary issues that users run into
+2. Depending on group size, work together or break into small groups to write our ideal documentation for a selected set of endpoints (e.g. a straighforward one like `/cat`, a complicated one like `/get` or recursive adds in `/add`, a weird one like `/p2p/listener/open`, and one that isn't implemented everywhere like `/mount`)
+3. Reconcile what’s in our ideal doc with the tools we have today and discuss how to improve our **tools** (expanding `go-ipfs-cmdkit` to fit more info, switching to OpenAPI, just writing docs in Markdown) and **processes** (how do we keep teams in sync/working together [unlike `interface-ipfs-core`] and keep the docs up to date).
+
+**Goal(s):**
+- A shared understanding of the ways in which the HTTP API docs fall down
+- An idea of what we want/need them to look like
+- A plan (or at least a set of tools) for getting to that ideal
```

Comment:
What if the one of the outputs is actually a work plan for a small team (~3) to revamp our http-api and respective docs. This way it would be immediately actionable and easy to ramp up new folks to do it.

Reply:
> What if the one of the outputs is actually a work plan for a small team

I like that a lot, but am concerned that we really won’t have the time to get that detailed. I tried to say this in the other comments on this PR, but my hope is that this session will mostly be about design and content; I’m not confident we will have time to do that *and* get into a detailed work plan. What if we make identifying an individual (or a few people) to lead and be responsible for implementing this a critical goal, and consider a work plan to be a bonus goal if we really have time?